lib/metrics.js

```diff
@@ -119,7 +119,7 @@ function Histogram(key, tags, host) {
     this.sum = 0;
     this.count = 0;
     this.samples = [];
-    this.percentiles = [0.75, 0.85, 0.95, 0.99];
+    this.percentiles = [0.95];
```

Comment:
can we get rid of this one too? this would reduce the cardinality even more. we can get this computed via datadog, we would just need to update the query used in the dashboards/monitors

Reply:
I'm pretty sure you cannot generate this with DD. My understanding is that this lib will accumulate X qty of metrics, then generate a histogram set based on those metrics then send to DD. It acts as a buffer and flushes itself every so often. So it actually helps reduce qty of metrics being sent, histogram of X set vs, every single event. (I was confused at how they were generating all these metrics from a single event... then dug further and realize it was a set of events)

If we just send avg I think there is potential to misrepresent data.

I looked into using a gauge type metric for this, but that also doesn't have the clarity we would want.

There might be somebody more familiar with this, but at this point I don't feel comfortable removing more metrics.

Reply:
Another idea would be to increase the buffer flush window?